src/style.css

```diff
@@ -0,0 +1,90 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Book.ttf");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-weight: bold;
+}
+
+.main {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
box-sizing: border-box; is duplicated, so make sense to move it to the `*{ box-sizing: border-box }`